pkg/ssh/ssh.go

```diff
@@ -702,3 +905,31 @@ func composeFileExists(host *models.Host) bool {
 	composeFileExists, _ := host.FileExists(utils.GetRemoteComposeFile())
 	return composeFileExists
 }
+
+func genesisFileExists(host *models.Host) bool {
+	genesisFileExists, _ := host.FileExists(filepath.Join(constants.CloudNodeConfigPath, constants.GenesisFileName))
+	return genesisFileExists
+}
+
+func getAvalancheGoConfigData(host *models.Host) (map[string]interface{}, error) {
+	// get remote node.json file
+	nodeJSONPath := filepath.Join(constants.CloudNodeConfigPath, constants.NodeFileName)
+	tmpFile, err := os.CreateTemp("", "avalanchecli-node-*.json")
+	if err != nil {
+		return nil, err
+	}
+	defer os.Remove(tmpFile.Name())
+	if err := host.Download(nodeJSONPath, tmpFile.Name(), constants.SSHFileOpsTimeout); err != nil {
```

Comment:
can we have a download that returns a byte slice?

Reply:
agree. we should have another func for this